scipy/stats/_continuous_distns.py

```diff
@@ -3891,34 +3917,53 @@ def f(theta):
             elif x0 == zeta:
                 return sc.gamma(1+1/alpha)*np.cos(xi)/np.pi/((1+zeta**2)**(1/alpha/2))
             else:
-                return levy_stable_gen._pdf_single_value_zolotarev(-x, alpha, -beta)
+                return levy_stable_gen._pdf_single_value_piecewise(-x, alpha, -beta)
         else:
             # since location zero, no need to reposition x for S_0 parameterization
             xi = np.pi/2
             if beta != 0:
-                warnings.warn('Density calculation unstable for alpha=1 and beta!=0.' +
-                              ' Use quadrature method instead.', RuntimeWarning)
-
                 def V(theta):
                     expr_1 = np.pi/2+beta*theta
-                    return 2. * expr_1 * np.exp(expr_1*np.tan(theta)/beta) / np.cos(theta) / np.pi
+                    return 2. * expr_1 * np.exp(expr_1*np.tan(theta)/beta) / np.cos(theta) / np.pi 
```

Comment:
These expressions are the same expect for the addition of a trailing whitespace in the new version? I don't think we want / need that.